src/ResourceManager/AzureBackup/Commands.AzureBackup/Helpers/ContainerHelpers.cs

```diff
@@ -40,31 +40,19 @@ internal class ContainerHelpers
 
         internal static AzureBackupContainerType GetContainerType(string customerType)
         {
-            CustomerType type = (CustomerType)Enum.Parse(typeof(CustomerType), customerType);
-
             AzureBackupContainerType containerType = 0;
 
-            switch (type)
+            if (string.Compare(customerType, CustomerType.DPM.ToString()) == 0)
```

Comment:
Instead of doing string comparision for enum, better use tryParseEnum method.
public static bool TryParse<TEnum>(string value, bool ignoreCase, out TEnum result) where TEnum : struct;
Actually I see Enum.Parse is being used at many places in powershell , this should bot be used . we shpuld use TryParse method to keep is compatibile with any new enum value coming in.